ts/features/itwallet/machine/credential/context.ts

```diff
@@ -3,13 +3,15 @@ import { StoredCredential } from "../../common/utils/itwTypesUtils";
 
 export type Context = {
   credentialType: CredentialType | undefined;
+  hardwareKeyTag: string | undefined;
```

Comment:
This was removed in  https://github.com/pagopa/io-app/pull/5861. Is this a merge artifact?